Services/Movies/Movies.API/Data/MoviesRepository.cs

```diff
@@ -93,6 +92,114 @@ public async Task<List<Genre>> GetGenres()
 
             return new Tuple<uint, uint, int, int>(yearMin, yearMax, lengthMin, lengthMax);
         }
+        
+        public async Task<string?> CreateMovie(CreateOrUpdateMovieRequest createOrUpdateMovieRequest)   
```

Comment:
Ovo je previse posla za repozitorijum. Mozda treba premestiti ovo u servis

Reply:
koja je folder struktura/konvencija za to? jesmo li imali negde slucaj da koristimo i servis

Reply:
ustv pogledacu kod tebe u identity-u :D